packages/lesmis/src/components/Chart.js

```diff
@@ -111,13 +116,19 @@ export const Chart = ({ name, viewContent, isLoading, isError, error, isEnlarged
   ) : (
     <>
       <Header>
-        <Title>{name}</Title>
+        <FlexStart>
+          <Title>{name}</Title>
+          {isFetching && !isLoading && getIsChartData(viewContent) && (
```

Comment:
Seems the same slightly complicated decision is being made in two places, so could be better to pull out a variable `isFullLoading`* within this component and then just pass that value through as a prop to the `ChartTable` as `isLoading`, and here use
```suggestion
          {isFetching && !isFullLoading && (
```
*could do with a better name

Reply:
I agree it does seem like they could be combined but I'm not quite sure how to combine the two decisions as the logic for them is actually different. Maybe I'm not seeing a way of doing it?
eg.
`isFetching && !isLoading && getIsChartData(viewContent)` vs `isLoading || (!getIsChartData(viewContent) && isFetching)`

Reply:
Would this work? It doesn't combine them all into one variable, which I also can't see a way to do, but pulls the logic to all be in one place https://github.com/beyondessential/tupaia/pull/2779 (and reuses the tricky decision of "fetching but from no data state, so mark as a full load")